### PR TITLE
Private-hand-48

### PR DESCRIPTION
**Pull request by erinyooey**
Task: 
Use two cards in player hand as private hand visible only to the player. (possiblly modify the dealer class in GardGames)

### CODE REVIEW

**Review by sophianicolas — COMMENTED** [review]
Nice work with creating the private hand function. It defines the function well and the tests for confirming the isKnown status as False and hand length seem to show that it holds up. 

I don't know if this will be needed later, but for a display it would be cool to have an area on the screen where the private hand can only be seen by the player.

**Comment by erinyooey** [comment]
Hi Sophia, thank you for your insight and feedback! That's a good recommendation :)

File: test/test_private.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from testing_base import *
+
+def test_twoCards():
```
Comment (Agwebberley) [inline test/test_private.py:3]: Would it be a good idea to add two players and test that the cards are only available to the correct player.  Is there currently any way to show the cards that are hidden?

**Review by Agwebberley — APPROVED** [review]

**Review by sophianicolas — APPROVED** [review]

File: test/test_private.py (hunk 1 above)
Comment (erinyooey) [reply test/test_private.py:3]: That is a good idea, I have made all the changes to ensure the cards are available to the correct player and that it displays the cards that are hidden. 
Screenshot 2024-04-05 at 9 37 09 PM

**Pull request merged**